Source/Turbo Navigator/TurboNavigatorDelegate.swift

```diff
@@ -20,6 +20,14 @@ public protocol TurboNavigatorDelegate: AnyObject {
     /// Optional. Respond to authentication challenge presented by web servers behing basic auth.
     /// If not implemented, default handling will be performed.
     func didReceiveAuthenticationChallenge(_ challenge: URLAuthenticationChallenge, completionHandler: @escaping (URLSession.AuthChallengeDisposition, URLCredential?) -> Void)
+    
+    /// Optional. Called before a form is submitted.
+    /// If not implemented, no action is taken.
+    func willSubmitForm(to url: URL)
+
+    /// Optional. Called when a form is submitted.
+    /// If not implemented, no action is taken.
+    func didSubmitForm(at url: URL)
```

Comment:
I would prefer that we carry the same start/finish language in these callback. As-is, it's not completely accurate. These callbacks are called after a form submission starts and after the form submission finishes.

Reply:
Good call. I went ahead and updated the names and documentation.

Reply:
Thanks!